tests/DbaaS/verifyDBaaS_test.js

```diff
@@ -47,6 +47,7 @@ Scenario(
     I.waitForVisible(dbaasPage.tabs.kubernetesClusterTab.addKubernetesClusterButtonInTable, 30);
     I.click(dbaasPage.tabs.kubernetesClusterTab.addKubernetesClusterButton);
     I.seeElement(dbaasPage.tabs.kubernetesClusterTab.modalWindow);
+    I.waitForElement(dbaasPage.tabs.kubernetesClusterTab.monitoringWarningLocator, 30);
```

Comment:
dbaasPage.tabs.kubernetesClusterTab.monitoringWarningLocator is not defined, dbaasPage.tabs.dbClusterTab.monitoringWarningLocator is. :) 
That's the reason why first test here fails https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-e2e-tests/detail/pmm2-dbaas-e2e-tests/736/tests/
The second one is unrelated.